### ISSUE TITLE
Synchronize IDOM version in other repos

### ISSUE THREAD

**Issue by Archmonger** [idom-team/idom#686]
### Current Situation

Currently, we don't have any CI to create automated PRs for changed IDOM versions. This results in repositories outside core to be perpetually out of date unless manually updated.

### Proposed Actions

Create a daily CI workflow to attempt to bump the IDOM version for the following repositories

- [ ] https://github.com/idom-team/idom-jupyter
- [ ] https://github.com/idom-team/django-idom
- [ ] https://github.com/idom-team/idom-dash
- [ ] https://github.com/idom-team/idom-plotly

**Comment by rmorshea** [comment]
Maybe related to #527

**Comment by rmorshea** [comment]
@acivitillo or @tony-maynard, this would greatly reduce the burden of regular maintenance across IDOM's repos.

**Comment by rmorshea** [comment]
I'm not quite sure what the best way to go about doing this would be, but my guess is that, each repo should have it's own action for synchronizing with `idom-team/idom`. If there's a way to trigger actions defined in the various downstream repos whenever a release happens, that would be ideal.

**Comment by Archmonger** [comment]
> Maybe related to #527

I don't believe so. This IDOM version sync would need to be separate from our "common files".

At least that's how I've seen this implemented in the past, the version sync process is very similar to how dependabot works. Except synching the version number to multiple different files.

**Comment by acivitillo** [comment]
I will be taking first the other issue I was assigned to

**Comment by Archmonger** [comment]
@rmorshea Since we're aiming for a monorepo, can we close this?

**Comment by rmorshea** [comment]
I'm not sure exactly how I want to handle it. The repo is set up for it, but maybe we limit it to just things that we want to include as namespace packages?
